src/main/scala/com/cognite/sdk/scala/v1/resources/raw.scala

```diff
@@ -200,6 +215,13 @@ class RawRows[F[_]](val requestSession: RequestSession[F], database: String, tab
       numPartitions: Int,
       limitPerPartition: Option[Int]
   )(implicit F: Applicative[F]): F[Seq[Stream[F, RawRow]]] =
+    _filterPartitionsF(filter, numPartitions, limitPerPartition)
+
+  private def _filterPartitionsF(
+      filter: RawRowFilter,
+      numPartitions: Int,
+      limitPerPartition: Option[Int]
+  ) =
```

Comment:
I'll load this into my IntelliJ later, but this is often because of importing `cats.implicits._` instead of `cats.syntax.all._`

On the other hand, I'm not sure we should piggy-back on `retrieveByExternalIds` for RAW - keys are approximately external ids, but not really. Does it make something else much simpler if we use `RetrieveByExternalIds` rather than just adding a RAW row-specific method named something else?

Reply:
not implementing `RetrieveByExternalIds ` makes everything easier 🚀 

I thought some tests might be reusable if I did it that way.... but no